src/player.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+'use strict';
+
+/**
+ * @typedef {Object} PrefPlayer
+ * @property {string} username Username
+ * @property {number} score Score
+ * @property {number} rating Rating
+ */
+
+export default class PrefPlayer {
+	private _username: string;
+	private _score: number;
+	private _rating: number;
+	private _change: number;
+
+	/** @constructor
+	 * @param {string} username - Username
+	 * @param {number} score - Score
+	 * @param {number} rating - Rating
+	 * @returns {object} PrefPlayer instance
+	 */
+	constructor(username: string, rating: number, score: number) {
+		this._username = username;
+		this._rating = rating;
+		this._score = score;
+		this._change = 0;
+
+		return this;
```

Comment:
"return this" is not needed in the ts constructors - it always returns instance of the class, just like in Java and C#

Reply:
Fixed.